docs/source/guides/libraries/java.rst

```diff
@@ -1,147 +1,39 @@
 Java Library
 ------------
 
-Prerequisites
-^^^^^^^^^^^^^
+The source code of the library, which can be used with Java and Kotlin projects can be found here: https://github.com/hyperledger/iroha-java/
 
-- Java 6
-- Gradle
+Please, read corresponding README.md file in aim to get more actual information
 
-Build Process
+Prerequisites
 ^^^^^^^^^^^^^
 
-Clone Iroha repository
-
-.. code-block:: shell
-
-  git clone https://github.com/hyperledger/iroha.git --depth=1
-  cd iroha
-
-.. note:: For the latest version checkout a ``develop`` branch by adding
-  ``-b develop`` parameter.
-
-Now we need to build our Java native library. Run
-``example/java/build_library.sh``, which will take care of it.
-
-.. code-block:: shell
-
-  cd example/java
-  ./build_library.sh
-
-.. note:: ``build_library.sh`` script creates a ``dist`` folder with
-  files, needed in :ref:`java-how-to-use` section.
+- Java 8 / Kotlin
+- Gradle
 
-.. _java-how-to-use:
 
 How to Use
 ^^^^^^^^^^
 
```

Comment:
Here you should mention that `iroha-java` library consists of 2 modules.
- `client` - client library for iroha. 
- `testcontainers` - library for integrational testing with Iroha

And you should mention that iroha-java can be imported not only in gradle, but using any preferred method from jitpack https://jitpack.io/#hyperledger/iroha-java